dawn/renderer/utils/utils.js

```diff
@@ -88,7 +88,12 @@ export class Logger {
       const { app } = require('electron'); // eslint-disable-line global-require
       path = app.getPath('desktop');
     }
-    this.log_file = fs.createWriteStream(`${path}/${Date.now()}-${processname}.log`, { flags: 'a' });
+    try {
+      fs.statSync(`${path}/Dawn`);
+    } catch (err) {
+      fs.mkdirSync(`${path}/Dawn`);
```

Comment:
It might be easier just to `try` the `mkdirSync` call and ignore the error if `err.code === 'EEXIST'`, removing the `statSync` call entirely.